src/pages/Group/Group.css

```diff
@@ -1 +1,277 @@
+html, body {
+    margin: 0;
+    padding: 0;
+}
 
+.group-hero {
+    /* display: flex; */
+    /* flex-direction: column; */
+    /* align-items: start; */
+    /* justify-content: center; */
+    padding-top: 80px;
+    padding-bottom: 80px;
+    padding-left: 70px;
+    padding-right: 70px;
+    /* width: 100vw; */
+    /* height: 100vh; */
+}
+
+.group-title {
+    text-decoration: underline;
+    background: linear-gradient(to right bottom, var(--sun-glow), var(--orange), var(--indian-red), var(--fuchsia-rose));
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: transparent;
+}
+
+.group-cards {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+.leftCardContainer {

```

Comment:
nit: keep the naming convention consistent! ex. leftCardContainer should be left-card-container